tools/config/update-configmap.sh

```diff
@@ -591,6 +591,18 @@ editEasHistoryPermissionJson=$(curl -sX GET "https://$SOAM_KC/auth/admin/realms/
   -H "Content-Type: application/json" \
   -H "Authorization: Bearer $TKN")
 
+echo Creating MANAGE_EAS_ASSESSMENT_KEYS_PERMISSION permission
```

Comment:
I'm not sure we want to add this permission - probably need to discuss with Marco

Reply:
The approach was to create separate permissions for managing ASSESSMENT KEYS, uploading ASSESSMENT RESULTS, and downloading XAM files. Currently, these permissions are mapped to EAS_ADMIN.